universe/config.json

```diff
@@ -40,7 +40,7 @@
                 "user": {
                     "description": "Executors will run as this user.",
                     "type": "string",
-                    "default": "root"
+                    "default": "nobody"
```

Comment:
might want to change it here too: https://github.com/mesosphere/spark-build/blob/master/history/universe/config.json#L33